js/lib/explore.min.js

```diff
@@ -117,8 +117,8 @@
 
 	/*!
 	 * Stickyfill -- `position: sticky` polyfill
-	 * v. 1.1.10 | https://github.com/wilddeer/stickyfill
-	 * Copyright Oleg Korsunsky | http://wd.dizaina.net/
+	 * v. 1.1.11 | https://github.com/18F/stickyfill
+	 * Copyright Brian Hedberg | 
```

Comment:
@meiqimichelle I have Brian Hedberg here instead of 18F because I couldn't figure out how to validate the module through my npmjs account without using my name :(